src/core/Window.h

```diff
@@ -58,39 +58,39 @@ class Window {
     Window() = default;
 
   private:
-    static bool KeyDown(int32_t dwScancode);
-    static bool KeyUp(int32_t dwScancode);
+    static bool KeyDown(int32_t scancode);
+    static bool KeyUp(int32_t scancode);
     static void AllKeysUp(void);
-    static void OnFullscreenChanged(bool bIsNowFullscreen);
+    static void OnFullscreenChanged(bool isNowFullscreen);
     static std::weak_ptr<Window> Context;
 
     void InitializeConfiguration();
     void InitializeControlDeck();
     void InitializeAudioPlayer();
     void InitializeLogging();
-    void InitializeResourceManager(const std::vector<std::string>& OTRFiles = {},
-                                   const std::unordered_set<uint32_t>& ValidHashes = {});
+    void InitializeResourceManager(const std::vector<std::string>& otrFiles = {},
+                                   const std::unordered_set<uint32_t>& validHashes = {});
     void InitializeWindowManager();
 
-    std::shared_ptr<spdlog::logger> Logger;
+    std::shared_ptr<spdlog::logger> mLogger;
     std::shared_ptr<Mercury>
-        Config; // Config needs to be after the Window because we call the Window during it's destructor.
-    std::shared_ptr<ResourceMgr> ResMan;
-    std::shared_ptr<AudioPlayer> APlayer;
-    std::shared_ptr<ControlDeck> ControllerApi;
+        mConfig; // Config needs to be after the Window because we call the Window during it's destructor.
+    std::shared_ptr<ResourceMgr> mResMan;
+    std::shared_ptr<AudioPlayer> mAudioPlayer;
+    std::shared_ptr<ControlDeck> mControlDeck;
 
-    std::string gfxBackend;
-    std::string audioBackend;
-    GfxRenderingAPI* RenderingApi;
-    GfxWindowManagerAPI* WmApi;
-    bool bIsFullscreen;
-    uint32_t dwWidth;
-    uint32_t dwHeight;
-    uint32_t dwMenubar;
-    int32_t lastScancode;
-    std::string Name;
-    std::string MainPath;
-    std::string BasePath;
-    std::string PatchesPath;
+    std::string mGfxBackend;
+    std::string mAudioBackend;
+    GfxRenderingAPI* mRenderingApi;
+    GfxWindowManagerAPI* mWmApi;
```

Comment:
going back and forth on this one, but i'll leave `mWindowManagerApi` as a suggestion